ScanbotSDKExamples/NET/ClassicComponent.Droid/ClassicComponent.Droid.csproj

```diff
@@ -9,12 +9,8 @@
         <ApplicationVersion>1</ApplicationVersion>
         <ApplicationDisplayVersion>1.0</ApplicationDisplayVersion>
         <AssemblyName>ClassicComponent.Droid</AssemblyName>
-        <ApplicationTitle>NET RTU UI</ApplicationTitle>
+        <ApplicationTitle>NET Classic Component</ApplicationTitle>

```

Comment:
@scan-mayank, consider that we may also have an example in the Barcode SDK following the same name, thus we would have a conflict.

Maybe we can have "Scanbot SDK.NET Classic Component" or "Scanbot SDK.NET Classic Component Example" or something that will make it more distinct.

I would also opt to keep ".NET" in the name, since we only leave out the dot in file names/paths because it can be confused for a file extension or hidden folder or something else.

Reply:
This also applies to the other projects/files updated as well.

Reply:
We can discuss and decide on an appropriate name.